spark/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -181,6 +181,18 @@ trait DeltaSQLConfBase {
       .booleanConf
       .createWithDefault(true)
 
+  val DELTA_INCLUDE_TABLE_ID_IN_FILE_INDEX_COMPARISON =
```

Comment:
This is currently unused. I guess it is a debug flag?

Reply:
Yes, for debugging/testing on a real cluster for now.